### PR TITLE
feat!(appengine): php72 -> php folder

### PR DESCRIPTION
**Pull request by ace-n**
~**Do not merge** until we have the appropriate docs changes prepared.~ See cl/355045170

### CODE REVIEW

**Review by bshaffer — CHANGES_REQUESTED** [review]
There are a LOT of instances of the path `appengine/php72/` in various sample `READMEs` and sample headers. We need to update them!

Examples:
 - https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/appengine/php72/storage/src/read_metadata.php#L21
 - https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/appengine/php72/auth/README.md#deploy-to-app-engine

**Comment by ace-n** [comment]
@bshaffer good catch! Fixed those via `sed`.

File: appengine/standard/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ## App Engine Standard samples
-Samples are now in appengine/php72](../php72) for the PHP 7.2 version of App Engine Standard.
+Samples are now in appengine/php](../php72) for the PHP 7.2 version of App Engine Standard.
```
Comment (bshaffer) [inline appengine/standard/README.md:2]: This also needs to be changed to `(../php)`

**Review by bshaffer — COMMENTED** [review]
Do we think `appengine/php` is where these should go? Since we've removed `php55` samples, I would advocate `appengine/standard` is clearer and more informative.

File: appengine/standard/README.md (hunk 1 above)
Comment (ace-n) [reply appengine/standard/README.md]: This was replaced with the `7.2` readme, which (now) has correct links.

**Review by bshaffer — APPROVED** [review]
LGTM!

**Comment by ace-n** [comment]
Updated `secrets` files and force-pushed changes, as requested by @bshaffer.

@grayside over to you. 🛩️

**Review by grayside — APPROVED** [review]

**Comment by bshaffer** [comment]
Adding do not merge as this is waiting on an internal docs change

**Pull request merged**